QualityCheckPostProcess/readme.md

```diff
@@ -0,0 +1,246 @@
+
+Here below are the confidence scores and their calculations for the largest groups of recordings which got Marosijo votes:
+
+| Score group | Score range | Vote type        | Amount  | Machine verified | Manually verified | Manually verified as valid | Manually verified as invalid | Confidence score     |
+|-------------|-------------|------------------|---------|------------------|-------------------|----------------------------|------------------------------|----------------------|
+| high        | 0.9 - 1.0   | Positive         | 701.904 | 435.550 (62.1%)  | 266.354 (37.9%)   | **227.186 (85.3%)**        | 39.168 (14.7%)               | **85.3%**            |
+| low         | 0.01 - 0.3  | Negative         | 20.486  | 16.518 (80.6%)   | 3.968 (19.4%)     | 380 (9.6%)                 | **3.588 (90.4%)**            | **90.4%**            |
+| zero        | 0           | Negative (super) | 33.313  | 13.122 (39.4%)   | 20.191 (60.6%)    | 137 (0.7%)                 | **20.054 (99.3%)**           | **99.3%**            |
+
+Bear in mind that the recordings that have been marked empty and given negative super votes, they are not Marosijo verified but inspected using another tool in the inspect_audio function of the GetRecordings module of the samromur-tools repo. Therefore, they do not have any scores but they have been proven to be truly empty and such recordings can safely be ignored while gathering recordings for training.
+
+The biggest limitation of a machine verification like this is that there have to be recordings on some score range which we cannot reliably tell if are good or bad based on the Marosijo scores. These scores are on the range of 0.301 to 0.899 and fall under a score group called *between*. The recordings in this group have been observed to be somewhat equally likely to be good or bad. Therefore, it was decided not to let scores in this range determine any votes. The reason why this is the *biggest* limitation of machine verification is because this range will inevitably be quite large (0.301 to 0.899) and therefore it contains a large amount of recordings, or 247.697 in our case. They all need human verification.
+
+## The tools
+This package contains the tools that were used to process the Marosijo output. It outputs a list of votes which are ready to be put into the Samrómur database.
+
+### generate_marosijo_votes.py
+This is the main utility for Marosijo score processing and vote generation. You give a path to a metadata file where the marosijo_score column has been filled with values, a path to a list of Montreal Forced Aligner unalignable recordings and some filenames for the output files.
+
+The output consists of two files:
```

Comment:
I only mention one of the files. I forgot to write about the statistics file. Should add a few line about it.